File: tests/language_completion/fixtures/rust/control_flow.rs
fn main() {
  let mut total = 0;
  while total < 5 {
    total = total + 1;
  }
  if total == 5 {
    println!("rust_control ready {}", total);
  } else {
    println!("rust_control bad {}", total);
  }
}

File: tests/language_completion/fixtures/dart/control_flow.dart
void main() {
  var count = 0;
  var total = 0;
  while (count < 5) {
    count = count + 1;
    if (count == 2) {
      continue;
    }
    total = total + count;
  }
  print("dart_control");
  print(total);
}
